bindings/python/src/torrent_handle.cpp

```diff
@@ -460,7 +460,7 @@ void bind_torrent_handle()
 #ifndef TORRENT_DISABLE_DHT
         .def("force_dht_announce", _(&torrent_handle::force_dht_announce))
 #endif
-        .def("scrape_tracker", _(&torrent_handle::scrape_tracker))
+        .def("scrape_tracker", _(&torrent_handle::scrape_tracker), arg("index") = -1)
```

Comment:
`torrent_handle::scrape_tracker` just takes one argument. The first arg in the Python exception is the `this` pointer, I think.